cli/src/main.rs

```diff
@@ -112,6 +112,9 @@ async fn main() -> Result<(), color_eyre::Report> {
     let config = iroha::combine_configs(&args)?;
     let telemetry = iroha_logger::init(&config.logger)?;
     if !config.disable_panic_terminal_colors {
+        // FIXME: it shouldn't be logged here; it is a part of configuration domain
```

Comment:
Stop-gap solution, specifically because it was a problem initially.

Reply:
Okay, i just put a comment to highlight a problem for future.